.travis.yml

```diff
@@ -157,14 +174,40 @@ script:
     fi;
   - if [ "${TRAVIS_OS_NAME}" == "osx" ]; then
       set -ex;
-      if [ "${RUN}" != "wheels" ]; then
+      if [ "${RUN}" == "unit" ]; then
         if [ "${PY}" == "3" ]; then
           python3 setup.py build_ext --inplace;
           python3 -m nose.core kivy/tests;
         else
           make;
           make test;
         fi;
+      elif [ "${RUN}" == "app" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]  && ([ "${TRAVIS_EVENT_TYPE}" == "cron" ] || [ "${TRAVIS_TAG}" != "" ]  || [[ "$TRAVIS_COMMIT_MESSAGE" =~ "[build app osx]" ]]); then
+          openssl aes-256-cbc -K $encrypted_675f1a0c317c_key -iv $encrypted_675f1a0c317c_iv -in ./kivy/tools/travis/id_rsa.enc -out ~/.ssh/id_rsa -d;
+          chmod 600 ~/.ssh/id_rsa;
+          echo -e "Host $SERVER_IP\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config;
+
+          git clone https://github.com/kivy/kivy-sdk-packager;
+          pushd kivy-sdk-packager/osx;
+          app_date=$(python -c "from datetime import datetime; print(datetime.utcnow().strftime('%Y%m%d'))");
+          git_tag=$(git rev-parse --short HEAD);
+          app_name="dev0.$app_date.$git_tag";
```

Comment:
Looks good overall, but here you may want to include the actual version. e.g. `app_ver=$(python -c 'import kivy; print(kivy.__version__)');` and `app_name="${app_ver/dev0/dev0.$app_date.$git_tag}";`. I'm not sure though if kivy is importable at that location?